### PR TITLE
Late review fixes for 5735

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: plugins/nexusoperations/frontend/fx.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	return &Config{
+		Enabled: coll.GetBoolProperty(dynamicconfig.FrontendEnableNexusAPIs, false),
+	}
+}
+
+func RegisterHTTPHandler(options HandlerOptions, logger log.Logger, router *mux.Router) {
+	h := nexus.NewCompletionHTTPHandler(nexus.CompletionHandlerOptions{
+		Handler: &completionHandler{
+			options,
+			options.MetricsHandler.Counter(metrics.NexusCompletionRequestPreProcessErrors.Name()),
```
Comment (rodrigozhou) [inline plugins/nexusoperations/frontend/fx.go:56]:
```suggestion
			HandlerOptions: options,
			preProcessErrorsCounter: options.MetricsHandler.Counter(metrics.NexusCompletionRequestPreProcessErrors.Name()),
```

File: plugins/nexusoperations/frontend/handler.go [hunk 2]
```diff
@@ -91,25 +76,27 @@ type HandlerOptions struct {
 
 type completionHandler struct {
 	HandlerOptions
+	preProcessErrorsCounter metrics.CounterIface
```
Comment (rodrigozhou) [inline plugins/nexusoperations/frontend/handler.go:79]: It doesn't need to be a member of `completionHandler` right? It's only used in `CompleteOperation` method.

Comment (bergundy) [reply plugins/nexusoperations/frontend/handler.go:79]: No good reason to recreate this every request.

File: plugins/nexusoperations/frontend/fx.go (hunk 1 above)
Comment (bergundy) [reply plugins/nexusoperations/frontend/fx.go:56]: 🤷

**Review by rodrigozhou — APPROVED** [review]

**Pull request merged**